Userland/Applications/Browser/Tab.h

```diff
@@ -44,6 +44,7 @@ class Tab final : public GUI::Widget {
         HistoryNavigation,
     };
 
+    Optional<URL> read_url(bool can_append_extension);
```

Comment:
Please use an `enum` for the parameter, so that the meaning can be understood at call sites.

So instead of this:
```cpp
	read_url(false);
```

We'd have something like:
```cpp
    url_from_location_bar(MayAppendTLD::No);
```